python/ecosystem.go

```diff
@@ -0,0 +1,73 @@
+package python
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/quay/claircore"
+	"github.com/quay/claircore/internal/indexer"
+)
+
+var scanners = []indexer.PackageScanner{&Scanner{}}
```

Comment:
by doing this you are sharing a single scanner between all go routines, this is okay?